main.go

```diff
@@ -22,6 +22,11 @@ func main() {
 		// plugin args
 		//
 
+		cli.BoolTFlag{
+			Name:   "dry-run",
+			Usage:  "if the plugin should pretend to upload but don't actually upload",
```

Comment:
my OCD triggered on this one... could we change it to `but doesn't actually upload` ?